rest/common.ts

```diff
@@ -103,8 +103,8 @@ export enum RESTJSONErrorCodes {
 	GuildAlreadyHasTemplate = 30031,
 	MaximumNumberOfApplicationCommandsReached,
 	MaximumThreadParticipants,
-
-	MaximumNumberOfNonGuildMemberBansHasBeenExceeded = 30035,
+	MaximumDailyApplicationCommandCreates,
```

Comment:
```suggestion
	MaximumDailyApplicationCommandCreatesReached,
```
think it's more understandable? i would do it even for the one above